src/Counters.jl

```diff
@@ -1,8 +1,9 @@
 using Base.Iterators: flatten
+using Random: rand
 
 flops(a, b, size) = prod(ind -> size[ind], a ∪ b, init=1)
 flops(a, b, size, keep) = prod(ind -> size[ind], flatten((a ∪ b, ∩(keep, a, b))), init=1)
 
 rank(a, b, size, keep) = length(symdiff(a, b) ∪ ∩(keep, a, b))
 
-removedsize(a, b, size, keep) = prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=1) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b)
\ No newline at end of file
+removedsize(a, b, size, keep) = prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=Float64(1)) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b)
```

Comment:
Why the `Float64`?
```suggestion
removedsize(a, b, size, keep) = prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=1) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b)
```

Reply:
I prefer to leave this function like it was and to transform to `Float` wherever we call to it.